packages/flutter_tools/test/general.shard/ios/ios_device_start_nonprebuilt_test.dart

```diff
@@ -343,8 +341,25 @@ IOSDevice setUpIOSDevice({
   );
 }
 
-class MockArtifacts extends Mock implements Artifacts {}
-class MockCache extends Mock implements Cache {}
 class MockXcode extends Mock implements Xcode {}
 class MockXcodeProjectInterpreter extends Mock implements XcodeProjectInterpreter {}
 class MockVmService extends Mock implements VmService {}
+
+class FakeEnvironmentArtifact extends ArtifactSet {
```

Comment:
Might be worth sharing this or making it a visibleForTesting artifact in cache.dart

Reply:
Is there somewhere else to put it?  I don't really want even more test fake code littering the real code.

Reply:
You could create a fakes.dart in the test utils folder, or at least something in the ios test folder.

Reply:
Should have answered my own question, I was tired of looking at it.

Reply:
I put it in the existing `fakes.dart` file.